examples_tests/39.DenoiserTonemapper/CommandLineHandler.hpp

```diff
@@ -46,7 +49,6 @@ Note there mustn't be any space characters!
 All file's resolutions must match!
 Also you must not put another data like comments or behaviour will be undefined, the app'll crash (TODO: not crash on parse failure)
 
-
 Description and usage: (TODO: Update these)
 
 ~OPENEXR_FILE: OpenEXR file containing various channels~
```

Comment:
update help text about file and channels

Reply:
done